### PR TITLE
Process new fields for author, date and subtitle. Create the news_header component

### PR DESCRIPTION
**Pull request by gremy**

### CODE REVIEW

File: kalagraphs.module [hunk 1]
```diff
@@ -223,24 +238,39 @@ function _kalagraphs_populate_fields(&$vars) {
...
+        $vars['subtitle'] = $value[$item]['summary'];
+      }
     }
   }
 
   // Provide a unique id attribute for the element that may or may not get used.
   $vars['id'] = Html::getUniqueId("kalagraphs-component-{$paragraph->field_kalagraphs_type->value}");
+  
+  // Force callback of the preprocess hook
+  \Drupal::moduleHandler()->invokeAll("preprocess_kalagraphs_{$paragraph->field_kalagraphs_type->value}", [&$vars]);
```
Comment (RobLoach) [inline kalagraphs.module]: Nice work on exposing the functionality to other modules to preprocess how the kalagraphs should define its variables.

**Review by RobLoach — APPROVED** [review]
Looks pretty good! @derekderaps, mind going through your own review of it? The usage of it is over at https://bitbucket.org/fairtradeusa/d8-fairtradeusa-org/pull-requests/166

File: kalagraphs.module (hunk 1 above)
Comment (hawkeyetwolf) [reply kalagraphs.module]: Seems that way, but we have different ideas in mind :) Hooks are dead! Take this bit out and then it's good to merge, the rest looks really good.

**Review by hawkeyetwolf — CHANGES_REQUESTED** [review]

**Review by hawkeyetwolf — APPROVED** [review]
Most excellent. The addition of author, date, and summary will be most helpful on other projects in the future. Thanks!

**Pull request merged**